src/common/schemas/TelegrafRequestPlugin.yml

```diff
@@ -1,14 +0,0 @@
-  oneOf:
-    - $ref: "./TelegrafPluginInputCpu.yml"
-    - $ref: "./TelegrafPluginInputDisk.yml"
-    - $ref: "./TelegrafPluginInputDiskio.yml"
-    - $ref: "./TelegrafPluginInputDocker.yml"
-    - $ref: "./TelegrafPluginInputFile.yml"
-    - $ref: "./TelegrafPluginInputKubernetes.yml"
-    - $ref: "./TelegrafPluginInputLogParser.yml"
-    - $ref: "./TelegrafPluginInputProcstat.yml"
-    - $ref: "./TelegrafPluginInputPrometheus.yml"
-    - $ref: "./TelegrafPluginInputRedis.yml"
-    - $ref: "./TelegrafPluginInputSyslog.yml"
-    - $ref: "./TelegrafPluginOutputFile.yml"
-    - $ref: "./TelegrafPluginOutputInfluxDBV2.yml"
```

Comment:
Want to remove these files as well, if possible?

Reply:
Yes, you are right. PR is updated.

Reply:
Sorry for misunderstood. Files are removed.